flixel/system/debug/console/Console.hx

```diff
@@ -119,6 +119,9 @@ class Console extends Window
 		{
 			if (FlxG.debugger.visible && FlxG.game.debugger.console.visible && e.keyCode == Keyboard.TAB)
 				FlxG.stage.focus = input;
+			#if windows
```

Comment:
nitpicky, I know but please put an empty line above here. I always try to put one empty after an if's block
```suggestion

			#if windows
```